draco/analysis/flagging.py

```diff
@@ -2281,11 +2293,14 @@ def fmask(f):
 
         # Solve to find a value of f that minimises the amount of data masked
         res = minimize_scalar(
-            fmask, method="golden", options={"maxiter": 20, "xtol": 1e-2}
+            fun=fmask,
+            bounds=(0, 1),
+            method="bounded",
```

Comment:
Glad this fixed the issue. Just curious: does the "bounded" method find similar values of `f` to "golden" in scenarios where both methods can be run? (i.e. scipy <= 3.8?). I also see below that it's not unusual for the optimization to fail to converge. Does "bounded" appear to have a similar convergence-failure rate to "golden"? I don't think this is that important, though. Looks great!

Reply:
Yes, in the testing that I did they end up finding the same value of `f`. I'm actually a little bit unsure about why that note about the optimization not converging is there - it seems to always converge in ~20 iterations.